apps/emqx_authn/src/emqx_authn.erl

```diff
@@ -126,3 +133,40 @@ get_enabled_authns() ->
 
 tally_authenticators(#{id := AuthenticatorName}, Acc) ->
     maps:update_with(AuthenticatorName, fun(N) -> N + 1 end, 1, Acc).
+
+%%------------------------------------------------------------------------------
+%% Data backup
+%%------------------------------------------------------------------------------
+
+-define(IMPORT_OPTS, #{rawconf_with_defaults => true, override_to => cluster}).
+
+import_data(RawConf) ->
+    AuthnList = maps:get(<<"authentication">>, RawConf, []),
```

Comment:
```suggestion
    AuthnList = maps:get(?CONF_NS_BINARY, RawConf, []),
```

Reply:
Thanks, changed all the occurrences in authn and authz